frontend/src/components/Navigator/__tests__/index.js

```diff
@@ -88,7 +90,54 @@ const initialData = {
 };
 
 describe('Navigator', () => {
-  beforeAll(async () => jest.useFakeTimers());
+  beforeAll(async () => {
```

Comment:
just moved the navigator and its context into a component, originally thought I had to add another context but I'm leaving it like this since it looks a little cleaner this way to my eyes